web/Controllers/LandlordController.cs

```diff
@@ -206,10 +206,12 @@ public async Task<IActionResult> ViewProperties(int? id = null)
         }
 
         var databaseResult = _propertyService.GetPropertiesByLandlord(id.Value);
-        var listOfProperties = databaseResult.Select(PropertyViewModel.FromDbModel).ToList();
+        var listOfProperties = databaseResult.Select(PropertyViewModel.FromDbModel).Skip((page - 1) * propPerPage)

```

Comment:
Could we have the pagination in the service please :)

Reply:
Sure, I forgot there was this page which needed re-paginating (every other paginated page is an Admin View, so easier to keep track of)